core/src/banking_stage.rs

```diff
@@ -1114,6 +1121,67 @@ impl BankingStage {
             .collect()
     }
 
+    fn generate_vote_packet_indexes(vers: &PinnedVec<Packet>) -> Vec<usize> {
+        vers.iter()
+            .enumerate()
+            .filter_map(|(index, ver)| {
+                if !ver.meta.discard {
+                    let tx = limited_deserialize(&ver.data[0..ver.meta.size]);
+                    if tx.is_ok()
```

Comment:
Can it just be this:
`let tx = limited_deserialize(&ver.data[0..ver.meta.size]).ok()?;`

Reply:
done! thanks